### PR TITLE
Fix incorrect progressbar infinite documentation

### PR DESCRIPTION
**Pull request by Jacalz**
### Description:

It was incorrectly referencing a value and max value while also commenting on internal API details.

### Checklist:

- [ ] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

### CODE REVIEW

File: widget/progressbarinfinite.go [hunk 1]
```diff
@@ -113,7 +113,7 @@ func (p *infProgressRenderer) start() {
 	}
 }
 
-// Stop the infinite progress goroutine and sets value to the Max
+// Stop the infinite progress bar background thread from to updating it
```
Comment (stuartmscott) [inline widget/progressbarinfinite.go:116]:
```suggestion
// Stop the infinite progress bar background thread from updating it
```

Or better yet:
```suggestion
// Stop the background thread from updating the infinite progress bar
```

Comment (stuartmscott) [reply widget/progressbarinfinite.go:116]: Note: this in an unexported function, so the doc is not developer-facing

**Review by stuartmscott — COMMENTED** [review]
Good call on using animation over goroutine/background thread

**Comment by andydotxyz** [comment]
Looks good, though I agree with Stuart about the changes.

File: widget/progressbarinfinite.go (hunk 1 above)
Comment (Jacalz) [reply widget/progressbarinfinite.go:116]: Oh, good suggestion. Indeed, that’s why I made sure to keep the internal details here 🙂

**Review by stuartmscott — APPROVED** [review]

**Pull request merged**